examples/wireless_wpa_psk.yml

```diff
@@ -0,0 +1,13 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- hosts: network-test
+  vars:
+    network_connections:
+      - name: wlan0
+        type: wireless
+        wireless:
+          ssid: "My WPA2-PSK Network"
+          key_mgmt: "wpa-psk"
+          # recommend vault encrypting the wireless password
+          # see https://docs.ansible.com/ansible/latest/user_guide/vault.html
+          password: "p@55w0rD"
```

Comment:
this example forgets to actually use the role (via `include_role` or similar).

Reply:
oops, my mistake